src/components/buttonWhiteIcon.js

```diff
@@ -0,0 +1,17 @@
+export const buttonWhiteIcon = (label, iconPath, onClick) => {
```

Comment:
```suggestion
export const iconButton = (label, iconPath, onClick) => {
```
Wystarczy nam bardziej generyczna nazwa tego komponentu

Reply:
Poprawione, dzięki!